.env.example

```diff
@@ -1,3 +1,4 @@
 DISCORD_TOKEN=
 DISCORD_PREFIX=
 API_URL=
+API_TOKEN=
```

Comment:
Before Deploying you need to adjust the Secret for the Bot as well, as the Env-Variables are getting adjusted with the API-Token

Reply:
oh yes 👍

Reply:
also the PR will need 

```yaml
           - name: API_TOKEN
              valueFrom:
                secretKeyRef:
                  name: api-token
                  key: API_TOKEN
```

https://github.com/EddieHubCommunity/EddieBot/blob/e5c08f456b6d603319fb7d90f199634c53c53d1e/kubernetes/deployment.yaml#L32-L36

I would add it myself but the PR is coming from your fork

Reply:
I have added the token to the Kubernetes cluster

Reply:
Awesome!